app/assets/stylesheets/layout.scss

```
@@ -82,6 +82,15 @@ body {
 @media (max-width: 768px) {
   .center-image {
     display: none;
+    flex: 0 0 100%;
+    max-width: 100%;
+  }
+}
+
+@media  (max-width: 768px) {
+  .center-text {
+    flex: 0 0 100%;
+    max-width: 100%;
   }
 }
```

You can merge these:
```suggestion
@media (max-width: 768px) {
  .center-image {
    display: none;
    flex: 0 0 100%;
    max-width: 100%;
  }

  .center-text {
    flex: 0 0 100%;
    max-width: 100%;
  }
}
```